src/kundera-rdbms/src/main/java/com/impetus/client/rdbms/query/RDBMSQuery.java

```diff
@@ -75,6 +75,7 @@ public RDBMSQuery(KunderaQuery kunderaQuery, PersistenceDelegator persistenceDel
             final KunderaMetadata kunderaMetadata)
     {
         super(kunderaQuery, persistenceDelegator, kunderaMetadata);
+        setMaxResults(0);
```

Comment:
I think `maxResults = 0` also mean unlimited results for Hibernate, doesn't it?
`Integer.MAX_VALUE` would limit you to an actual value so it wouldn't be actually unlimited.